src/Microsoft.Security.Utilities.Core/PotentialSecurityKeys/SEC000_003_Unclassified32CharacterString.cs

```diff
@@ -1,102 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System;
-using System.Collections.Generic;
-
-namespace Microsoft.Security.Utilities
-{
-    public class Unclassified32CharacterString : RegexPattern
-    {
-        public const string AzureContainerRegistryLegacyKey = nameof(AzureContainerRegistryLegacyKey);
-        public const string AadClientAppLegacyCredentials = nameof(AadClientAppLegacyCredentials);
-
-        /// <summary>
-        /// Detect 32-character Azure Active Directory client application legacy credentials.
-        /// The generated key is a 32-character string that contains alphanumeric characters
-        /// as well as symbols from the set: .=\-:[_@\*]+?
-        /// </summary>
-        public Unclassified32CharacterString()
```

Comment:
>Unclassified32CharacterString [](http://example.com/codeflow?start=15&length=29)

Problem #1 in the strange test code paths Ross found suspicious: the test data generated by this rule was not in strict conformance across all its logic.

Basically, we have no general testing of the `UnclassifiedPotentialSecurityKeys` patterns list, which I could go slam into some other bucketing test. I am deferring that work for now.

For now, I'm ripping this rule out because it's noisy and not useful. The intent here was to capture an ancient variation of an generated AAD SPN secret, then the rule was renamed as a generic `Unclassified32CharacterString` because it fired too frequently and really it should have been deleted entirely. If there's an analysis here, it will argue its way back in.